jgrapht-core/src/test/java/org/jgrapht/alg/scoring/AlphaCentralityTest.java

```diff
@@ -0,0 +1,288 @@
+/*
+ * (C) Copyright 2018-2018, by Pratik Tibrewal and Contributors.
+ *
+ * JGraphT : a free Java graph-theory library
+ *
+ * This program and the accompanying materials are dual-licensed under
+ * either
+ *
+ * (a) the terms of the GNU Lesser General Public License version 2.1
+ * as published by the Free Software Foundation, or (at your option) any
+ * later version.
+ *
+ * or (per the licensee's choosing)
+ *
+ * (b) the terms of the Eclipse Public License v1.0 as published by
+ * the Eclipse Foundation.
+ */
+package org.jgrapht.alg.scoring;
+
+import org.jgrapht.alg.interfaces.*;
+import org.jgrapht.graph.*;
+
+import junit.framework.*;
+
+/**
+ * Unit tests for AlphaCentrality
```

Comment:
It looks like you took the example graphs verbatim from PageRankTest.  Shouldn't you be also adding some new graphs which demonstrate why one might want to use this measure?  Also, calling on the same input graph more than once with different parameters is a good way to demonstrate their effect.  (Unit tests serve an additional purpose of implicit documentation.)

Reply:
I used the graph mentioned in this [page](https://www.sci.unich.it/~francesc/teaching/network/katz.html). It turns out that the implementation of the [PageRankTest graph](https://www.sci.unich.it/~francesc/teaching/network/pagerank) is same as that of I tested. I think the difference in the outputs of both the graphs demonstrates the difference in the implementations of both the graphs, isn't it?